spec/acceptance/locales_spec.rb

```diff
@@ -12,7 +12,7 @@
   context 'when triggering puppet simple string error' do
     # 'service_enabled' being set to false can cause random failures in Debian 9
     let(:os_variant) do
-      if fact('operatingsystem') =~ %r{Debian} && fact('operatingsystemrelease') =~ %r{^9\.}
+      if os[:release] =~ %r{Debian} && os[:release] =~ %r{^9\.}
```

Comment:
When using relesase it returns a longer string like 9.87.6655 
An easy way to avoid confusion is something like this:
os[:release].to_i == '9'
os[:release]  returns an integer it will never match 'Debian'